cinome/cinomedjango/cinomeapp/models.py

```diff
@@ -1,7 +1,59 @@
+from django.core.validators import MinLengthValidator, MinValueValidator, MaxValueValidator
 from django.db import models
 
+
 class Test(models.Model):
-   title = models.CharField(max_length=100)
+    title = models.CharField(max_length=100)
+
+    def _str_(self):
+        return self.title
+
+
+class User(models.Model):
+    email = models.EmailField()
+    password = models.CharField(max_length=255, validators=[MinLengthValidator(4)])
+    username = models.CharField(max_length=32, validators=[MinLengthValidator(4)], unique=True)
+    image_url = models.FilePathField(blank=True)
+    is_email_confirmed = models.BooleanField(default=False)
+
+
+class UserFriends(models.Model):
+    user = models.ForeignKey(User, on_delete=models.RESTRICT)
+    friend = models.ForeignKey(User, on_delete=models.RESTRICT, related_name="friend_relation")
```

Comment:
Do we really need to create such models without any additional logic? I see a lot of navigation properties throughout the whole file. Maybe we could somehow specify this table should also be created? I believe there should be a similar approach to the one we used in EF.

Reply:
Yes, you're right. I'll fix it.